src/spider/worker/worker.cpp

```
@@ -50,29 +50,62 @@ constexpr int cTaskErr = 5;
+        boost::program_options::variables_map variables;
+        boost::program_options::store(
+                // NOLINTNEXTLINE(misc-include-cleaner)
+                boost::program_options::parse_command_line(argc, argv, desc),
+                variables
+        );
+
+        if (!variables.contains("host") && !variables.contains("storage_url")
+            && !variables.contains("libs"))
+        {
+            std::cout << cUsage << "\n";
+            std::cout << desc << "\n";
```

cppcheck (2.10-2)

[performance] Variable 'id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)